### DISCUSSION TITLE
Excerpts from Cardew's Treatise for neoscore Formalization

### DISCUSSION THREAD

**Discussion by jgarte** [DigiScore/neoscore#45] [Ideas]
wdyt

https://blogthehum.com/2016/02/29/cornelius-cardews-treatise-1963-67/

![cornelius_cardew_-_treatise-31](https://user-images.githubusercontent.com/47760695/175179705-fc10f964-f1a5-4664-8486-6ec16b745ee7.jpeg)

![2022-06-22-192244_1029x591_scrot](https://user-images.githubusercontent.com/47760695/175181955-331c38dd-863b-4226-871d-09ebd40aca4d.png)

**Comment by ajyoon** [comment]
Most of Treatise is easily implementable with neoscore, and I believe a couple individual pages can be safely excerpted and reproduced under UK Fair Dealing law.
